api/rs/slint/mcu.md

```diff
@@ -38,28 +38,21 @@ features = ["compat-0.3.0", "unsafe-single-threaded", "libm"]
 slint-build = "0.3"
 ```
 
-The default features of the `slint` create are tailored towards hosted environments and includes the "std" feature. In bare metal environments,
+The default features of the `slint` crate are tailored towards hosted environments and includes the "std" feature. In bare metal environments,
 you need to disable the default features.
 
-Three features are selected:
+In the snippet above, three features are selected:
 
- * `compat-0.3.0`: You need to select this feature when disabling the default features. See [this blog post](https://slint-ui.com/blog/rust-adding-default-cargo-feature.html)
-   for a detailed explanation.
+ * `compat-0.3.0`: We select this feature when disabling the default features. For a detailed explanation see our blog post ["Adding default cargo features without breaking Semantic Versioning"](https://slint-ui.com/blog/rust-adding-default-cargo-feature.html).
  * `unsafe-single-threaded`: Slint internally uses Rust's [`thread_local!`](https://doc.rust-lang.org/std/macro.thread_local.html) macro to store global data.
-   This feature is only available in the Rust Standard Library (std), which is not available in bare-metal environments. As a fallback, the `unsafe-single-threaded`
-   feature will change Slint to use unsafe static for storage. By setting this feature, you guarantee not to use Slint API from a thread other than the main thread,
-   or from interrupt handlers.
- * `libm`: The Rust Standard Library (std) provides traits and functions for floating point arithmetic. Without `std`, this feature enables the use of the
-   [libm](https://crates.io/crates/libm) crate to substitute this functionality.
+   This feature is only available in the Rust Standard Library (std), but is not available in bare metal environments. As a fallback, the `unsafe-single-threaded`
+   feature changes Slint to use unsafe static for storage. This way, you guarantee to use Slint API only from the main thread, or from interrupt handlers.
```

Comment:
```suggestion
   This macro is only available in the Rust Standard Library (std), not in bare metal environments. As a fallback, the `unsafe-single-threaded`
   feature changes Slint to use unsafe static for storage. By selecting this feature, you must ensure that you only use
   the Slint API from a single thread and not from interrupt handlers.
```

The sense of the sentence was changed: It is **not** possible to use slint from interrupt handlers

Reply:
I remove the redundant "available" (well spotted).

Well spotted about the change of meaning!